tox.ini

```diff
@@ -40,11 +40,10 @@ atomic = true
 force_grid_wrap = 0
 include_trailing_comma = true
 lines_after_imports = 2
-lines_between_types = 1
 multi_line_output = 3
 not_skip = __init__.py
 use_parentheses = true
-known_first_party = kw.platform
+known_first_party = kw.platform,dateutil,pytest,webob
```

Comment:
I think I did this because `isort` triggered by `pre-commit` doesn't read configuration, while manually running `isort` respects configuration. This was a very bad workaround (somehow automatically generated by seed-isort-config which I later removed as it expects `setup.py` and thus doesn't work with poetry), I'm probably gonna remove the isort configuration, it seems that for such a simple library, its incredibly hard to configure it properly.

I would recommend using the default configuration everywhere, in every repository, it's pain to have different configuration for each repository, different configuration in text editors, and having pre-commit hooks which don't properly read the configuration.

Reply:
OK, Alex helped me with fixing this. The `seed-isort-config` is necessary when using isort in pre-commit, so we added it there (it didn't work before because we were using old version, I should've checked it).

So now it works as expected, but I'm still surprised we need all these libraries just to sort imports correctly :-)